routes/userRoutes.js

```diff
@@ -10,14 +10,22 @@ router.post('/:token/activate', userController.activateAccount);
 router.get('/:username/following', userController.showFollowingUsers);
 router.get('/:username/followers', userController.showFollowerUsers);
 router.post('/:username/follow', userAuth.isAuthenticated, userController.followUser);
+
+router.get('/:username', userController.showUserProfile);
+router.get('/:username/projects', userController.showUserProjects);
+router.get('/:username/projects/public', userController.showUserProjectsPublic);
+router.get('/:username/projects/draft', userController.showUserProjectsDraft);
```

Comment:
for drafts, applied >> these are private and will only be displayed to the user themselves right?
if that is the case, will need authorization here

Reply:
this one, can you help me put when you help me do the userIsAuthorised?